boards/arm/teensy4/teensy41.dts

```diff
@@ -11,7 +11,7 @@
 	reg = < 0x402a8000 0x4000 >, < 0x60000000 0x800000 >;
 	/* WINBOND flash memory*/
 	w25q64jvxgim: w25q64jvxgim@0 {
-		compatible = "winbond,w25q64jvxgim", "jedec,spi-nor";
+		compatible = "nxp,imx-flexspi-nor";
 		size = < 8388607 >;
```

Comment:
@danieldegrasse just to be clear, I don't own any of these boards, but I think that this `size` and the one above are also wrong, the numbers make no sense to me (and this block also has the wrong style). LMK if you want me to change them, I'm mostly concerned with getting this off my way because it breaks the CI on an unrelated PR that is touching the flash tests and triggering a test for this board which does not build. For the board, I suspect that the flash support here either never worked or broke somewhere along the way and no one noticed.

Reply:
I also don't have any of these boards (I believe @dleach02 may have one). If we want to fix this short term, I would prefer to avoid using the `nxp,imx-flexspi-nor` compatible, as that will enable NXP's FlexSPI NOR driver with this flash chip. Since I can't validate that the driver actually functions with this flash chip, I'm hesitant to enable it.

Maybe we can set the compatible to `winbond,w25q64jvxgim`? This way, flash driver support won't be enabled, but CI should still pass.

Reply:
Sure, though there's no `winbond,w25q64jvxgim` binding, I think what's going to happen is that this is going to fix the build, but will leave the flash access from the code with the flash API equally broken. Good enough for CI I guess.

Reply:
Yeah, I don't think this will enable flash on the board. I confirmed @dleach02 has a teensy board, so we may be enable to test this and see if the flash can be enabled with this driver.

Reply:
Thanks, may be worth reviewing that config from scratch, those numbers looks odd. Quite literally in fact (8388607).

Reply:
The numbers are derived from the used flash chips. For the teensy4.1 the winbond datasheet declares the flash chip to have size of "32,768 programmable pages of 256-bytes each" which would result in the given 8388608 B. I'm just unsure about why this value minus 1.